src/System.ComponentModel.TypeConverter/tests/SampleClasses.cs

```diff
@@ -82,13 +82,7 @@ public MyTypeListConverter(Type[] types)
         }
     }
 
-#if FUNCTIONAL_TESTS
-    [TypeConverter("System.ComponentModel.Tests.BaseClassConverter, System.ComponentModel.TypeConverter.Tests, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null")]
```

Comment:
Yes, that is why I pointed it out, to see if I can get any thoughts on that one :) thanks @tarekgh